package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "copy-as-rtf",
   "main": "./lib/copy-as-rtf",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Could you please leave this change out of this PR? I rather let the `apm publish minor` do the version bump in a separate commit.